packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -17,7 +17,11 @@ export type PopperPlacementType =
   | 'top-start'
   | 'top';
 
-export interface PopperProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> {
+export interface PopperProps
+  extends Omit<
+    React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
+    'children'
+  > {
```

Comment:
What do you think of this approach instead? To match #17835

```suggestion
export interface PopperProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> {
  ref?: React.Ref<HTMLDivElement>;
```